jigna/web_server.py

```diff
@@ -50,7 +54,10 @@ def send_event(self, event):
         message_id = -1
         data = json.dumps([message_id, jsonized_event])
         for socket in self._active_sockets:
-            socket.write_message(data)
+            if main_thread:
+                socket.write_message(data)
+            else:
```

Comment:
Maybe add a comment here that tornado does not support calling `socket.write_message` from a thread and a link to the article you mentioned on the PR comment?